src/AVFoundation/AVCompat.cs

```diff
@@ -128,7 +130,7 @@ public virtual AVMetadataItem[] ChapterMetadataGroups (NSLocale forLocale, AVMet
 	partial class AVAssetTrack {
 
 		[Obsolete ("Use 'GetAssociatedTracks'.")]
```

Comment:
@rolfbjarne  @tj-devel709  all this method that have not implemented exception, should we just wrap them around a !NET?

Reply:
Add xamcore5